components/ui/TextField.tsx

```diff
@@ -0,0 +1,13 @@
+import styled from "@emotion/styled";
+
+
+const TextField = styled.input`
+  width: 30px;
+  border-radius: 25px;
+  font-size: 16px;
+  color: #303A52;
```

Comment:
Please use color from CSS Custom properties specify in `GlobalStyles`. You can see how to use it online or on other files.